roles/zabbix_agent/tasks/Windows_conf.yml

```diff
@@ -2,7 +2,14 @@
 - name: "Set Log File Info"
   ansible.builtin.set_fact:
     zabbix_agent_logfile: "{{ zabbix_agent_win_logfile is defined | ternary(zabbix_agent_win_logfile, zabbix_agent2_win_logfile) | default(_win_logfile) }}"
-  when: zabbix_agent_logfile is undefined
+
+- name: "Set Include Path Info"
+  ansible.builtin.set_fact:
+    zabbix_agent_include: "{{ zabbix_agent_win_include is defined | ternary(zabbix_agent_win_include, zabbix_agent2_win_include) | default(_win_include) }}"
```

Comment:
See the latest changes.  Support for `zabbix_agent2_win_include` has been removed with the release of 3.0.0

Reply:
In template of zabbix_agent.conf, the parameter `Include=` **will be set** because the variable `zabbix_agent_include` is defined in the main.yml script.

https://github.com/ansible-collections/community.zabbix/blob/45ed88b2abbaffb8869bcb102ac2ce5ed6694769/roles/zabbix_agent/tasks/main.yml#L27-L29

https://github.com/ansible-collections/community.zabbix/blob/45ed88b2abbaffb8869bcb102ac2ce5ed6694769/roles/zabbix_agent/templates/agent.conf.j2#L69

So, for Windows, if the variable `zabbix_agent_include` was not explicited defined, the parameter `Include` will be set to `/etc/zabbix/zabbix_agentd.d` because of the code below.

https://github.com/ansible-collections/community.zabbix/blob/45ed88b2abbaffb8869bcb102ac2ce5ed6694769/roles/zabbix_agent/vars/agent_vars.yml#L3

Reply:
Maybe I misunderstood your comment. Do you just want to unify the parameter for Zabbix_agent and Zabbix_agent2?